### ISSUE TITLE
[jsk_apc_2016_baxter_apc] robot self filter does not work properly

### ISSUE THREAD

**Issue by knorth55** [start-jsk/jsk_apc#1456]
Because `robot_description` is modified, `robot_self_filter` does not work properly so that it failed to verify object in hand.
Upper: before self filtered
Lower: after self filtered 
![before_filter](https://cloud.githubusercontent.com/assets/9300063/15222662/cb140d62-18ab-11e6-8be7-06bf9675054c.png)
![after_filter](https://cloud.githubusercontent.com/assets/9300063/15222680/dec6847a-18ab-11e6-899c-dbc016240dde.png)

**Comment by knorth55** [comment]
@wkentaro do you know how to visualize self filtered region?

**Comment by knorth55** [comment]
robot self filter works properly with stl model?

**Comment by knorth55** [comment]
i convert stl models to collada models, but it does not work

**Comment by knorth55** [comment]
@pazeshun can you cherry-pick the commit and try it on gripper-v2 branch?

**Comment by pazeshun** [comment]
@knorth55
ok, I'll try it for Saturday demo

**Comment by wkentaro** [comment]
This seems to be introduced by new gripper model, so I change the assignee to @pazeshun .

**Comment by pazeshun** [comment]
I made a new branch for debugging from gripper-v2 branch and cherry-picked a part of #1458 
https://github.com/pazeshun/jsk_apc/tree/fix-self-filter-2016
In debugging, I realized that robot_self_filter works when I comment out `right_gripper_base` and `right_gripper_vacuum_pad_base` in self_filter.yaml.
![screenshot from 2016-05-14 11 31 20](https://cloud.githubusercontent.com/assets/14994939/15265768/068b21be-19c9-11e6-9add-fb20bffb9ded.png)

**Comment by pazeshun** [comment]
I changed collision meshes of `right_gripper_base` and `right_gripper_vacuum_pad_base` from STL data to URDF boxes.
Then robot_self_filter worked.
![screenshot from 2016-05-14 17 28 28](https://cloud.githubusercontent.com/assets/14994939/15267358/de49df42-19fa-11e6-9c42-a909cef92567.png)
I'll make suitable collision meshes of these two links using URDF boxes and cylinders.

**Comment by pazeshun** [comment]
Also, I realized that padding of grippers in self_filter.yaml of 2015 is too big for new right gripper.
![screenshot from 2016-05-14 17 57 48](https://cloud.githubusercontent.com/assets/14994939/15267455/a0ac93ac-19fd-11e6-9603-981ce3dd865d.png)
In the above picture, I displayed visual and collision links at the same time. I can say that visual meshes are bigger than collision meshes in left gripper.
I think padding of right gripper should be less than left gripper, because visual meshes of right gripper will be the same size as, or smaller than collision meshes.
In fact, /extract_indices_right_hand/points_filtered lost many points when padding is the same value as left gripper.
![screenshot from 2016-05-14 18 14 43](https://cloud.githubusercontent.com/assets/14994939/15267519/c4d4d4c2-19ff-11e6-927d-71ed4acac77d.png)
